scripts/jq/1.5-239278fd/.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: generic
+
+matrix:
+  include:
+    - os: osx
+      osx_image: xcode8
+      compiler: clang
+    - os: linux
+      compiler: clang
+      sudo: false
+      addons:
+        apt:
+          sources: [ 'ubuntu-toolchain-r-test' ]
+          packages: [ 'libstdc++-4.9-dev', 'libssl-dev' ]
```

Comment:
Minor, but I think you can remove `libssl-dev` here, right? That was only needed for wget and not jq, correct?